tests/test_search.py

```diff
@@ -1440,6 +1440,26 @@ def test_aggregations_filter(client):
             assert res["results"][1]["extra_attributes"] == {"age": "25"}
 
 
+@pytest.mark.redismod
+@skip_ifmodversion_lt("2.10.05", "search")
+def test_aggregations_add_scores(client):
+    client.ft().create_index(
+        (
+            TextField("name", sortable=True, weight=5.0),
+            NumericField("age", sortable=True),
+        )
+    )
+
+    client.ft().client.hset("doc1", mapping={"name": "bar", "age": "25"})
```

Comment:
See my comment above. In this case should be just `client` instead of `client.ft().client`